packages/ipfs-gateway-race/docker/docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: '3.6'
+
+services:
+  ipfs0:
+    container_name: ipfs0
+    image: ipfs/go-ipfs:v0.10.0 # update this when go-ipfs M1 macs https://github.com/ipfs/go-ipfs/issues/8645
```

i think you can update this now
```suggestion
    image: ipfs/go-ipfs:v0.13.0
```